ansible/roles/letsencrypt/templates/project-le-ssl.conf

```diff
@@ -0,0 +1,31 @@
+<IfModule mod_ssl.c>
```

Comment:
issue this should copy the non-ssl one then use ansible's lineinfile to add the ssl stuff.